### ISSUE TITLE
In recent versions after 3.2.5-linux1 `github` command is broken

### ISSUE THREAD

**Issue by sarim** [shiftkey/desktop#919]
### The problem

I'm using the deb files, tried both deb from github release and from your repo. The problem is github command does nothing. Github Desktop doesn't open.

/usr/bin/github script essentially runs this command: `ELECTRON_RUN_AS_NODE=1 /usr/lib/github-desktop/github-desktop /usr/lib/github-desktop/resources/app/cli.js`, but sends output to /dev/null, so you couldn't see the error. Manually running it.

```sh
$ ELECTRON_RUN_AS_NODE=1 /usr/lib/github-desktop/github-desktop /usr/lib/github-desktop/resources/app/cli.js
ERR! Desktop command line interface not currently supported on platform linux
Error: Desktop command line interface not currently supported on platform linux
    at t (/usr/lib/github-desktop/resources/app/cli.js:1:26473)
    at Object.handler (/usr/lib/github-desktop/resources/app/cli.js:1:26160)
    at h (/usr/lib/github-desktop/resources/app/cli.js:1:28549)
    at e (/usr/lib/github-desktop/resources/app/cli.js:1:28693)
    at /usr/lib/github-desktop/resources/app/cli.js:1:28764
    at /usr/lib/github-desktop/resources/app/cli.js:1:28783
    at Object.<anonymous> (/usr/lib/github-desktop/resources/app/cli.js:1:28808)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1296:10)
    at Module.load (node:internal/modules/cjs/loader:1096:32)
Commands:
  clone <url|slug>  Clone a repository
  help [command]    Show the help page for a command
  open <path>       Open a git repository in GitHub Desktop

Run github help <command> for details about each command

```

The issue is somehow this commit https://github.com/shiftkey/desktop/commit/70957fd90acbe786fdec19ca318994009623580b didn't get included in recent builds.

Confirmation: Since cli.js is uglified, lets read the js.map with 

```sh
jq -r '.sourcesContent[22]' /usr/lib/github-desktop/resources/app/cli.js.map
```
outputs
```js
import * as ChildProcess from 'child_process'

export function openDesktop(url: string = '') {
  const env = { ...process.env }
  // NB: We're gonna launch Desktop and we definitely don't want to carry over
  // `ELECTRON_RUN_AS_NODE`. This seems to only happen on Windows.
  delete env['ELECTRON_RUN_AS_NODE']

  url = 'x-github-client://' + url

  if (__DARWIN__) {
    return ChildProcess.spawn('open', [url], { env })
  } else if (__WIN32__) {
    // https://github.com/nodejs/node/blob/b39dabefe6d/lib/child_process.js#L565-L577
    const shell = process.env.comspec || 'cmd.exe'
    return ChildProcess.spawn(shell, ['/d', '/c', 'start', url], { env })
  } else {
    throw new Error(
      `Desktop command line interface not currently supported on platform ${process.platform}`
    )
  }
}
```

HERE look, the `if (__LINUX__)` block is missing. 

The linked commit is needed. Also if you can please check if any other commit got lost in merge/rebase operations.

Also github command shouldn't just fail silently, but that's another issue. After fixing this issue. If you're interested, I can try rewriting /usr/bin/github script.

### Release version

3.2.7-linux2

### Operating system

Ubuntu 22.04

**Comment by shiftkey** [comment]
@sarim thanks for the report - I suspect this was introduced as part of the tooling changes but I haven't yet been able to reproduce it on my end: https://github.com/shiftkey/desktop/issues/903#issuecomment-1670096044

**Comment by shiftkey** [comment]
Looks like I messed up cutting the release, as the version that we shipped does indeed omit that `__LINUX__` code path:

https://github.com/shiftkey/desktop/blob/c4fa855efa39b65ccbb57b185a40857f7f3402e5/app/src/cli/open-desktop.ts#L11-L21

That should be easy enough to fix, and I'll re-check when I cut this next release.

**Comment by shiftkey** [comment]
This has been fixed in [`3.2.8-linux1`](https://github.com/shiftkey/desktop/releases/tag/release-3.2.8-linux1)
